pkg/api/schemav2/schemav2.proto

```diff
@@ -0,0 +1,190 @@
+syntax = "proto3";
```

Comment:
what about creating an object schema?

Reply:
Resolved in 
https://github.com/codenotary/immudb/pull/1546